tests/modules/assets/resources/test_assets.py

```diff
@@ -19,10 +20,9 @@
 def test_get_asset_not_found(flask_app_client, researcher_1):
     import uuid
 
-    asset_utils.read_asset(flask_app_client, None, str(uuid.uuid4()), 401)
+    asset_utils.read_asset(flask_app_client, None, str(uuid.uuid4()), 404)
     asset_utils.read_asset(flask_app_client, researcher_1, str(uuid.uuid4()), 404)
-    # TODO, this is what the test did previously, does this make sense?
```

Comment:
@hwindsor Yes, the test was testing what happened when we get an invalid uuid (I'm
making it clearer by changing "wrong-uuid" to "invalid-uuid").  The
original problem is when we're given an invalid uuid, the path in
`app/modules/assets/resources.py` won't match because it only matches
uuids.  This used to go to `app.modules.frontend.views_frontend.home`
and return 200 and something completely unexpected.

More in commit db84f6c90